pkg/jobs/jobspb/jobs.proto

```diff
@@ -493,7 +493,8 @@ message TypeSchemaChangeProgress {
 
 // TypeSchemaChangeDetails is the job detail information for the new schema change job.
 message NewSchemaChangeDetails {
-  repeated cockroach.sql.schemachanger.scpb.Target targets = 1;
+
+  cockroach.sql.schemachanger.scpb.TargetState target_state = 5 [(gogoproto.nullable) = false];
```

Comment:
do we need to `reserve 1,2,3`?

Reply:
No, these jobs don't exist yet out in the wild. We'll do a tag ID cleanup pass before the release, this is liable to change again until then.